src/content/LibraryContent.tsx

```diff
@@ -10,6 +10,13 @@ export const libraries: Library[] = [
     repository: 'https://github.com/process-analytics/bpmn-visualization-js',
     type: 'TypeScript',
   },
+  {
+    name: 'BPMN Visualization - R package',
+    description:
+      'An experimental implementation to embed the BPMN Visualization library in an R package',
```

Comment:
Add a final dot for consistency
```suggestion
      'An experimental implementation to embed the BPMN Visualization library in an R package.',
```